devito/builtins.py

```diff
@@ -7,21 +7,40 @@
 
 import devito as dv
 
-__all__ = ['assign', 'smooth', 'norm', 'sumall', 'inner', 'mmin', 'mmax']
+__all__ = ['assign', 'smooth', 'gaussian_smooth', 'initialize_function', 'norm',
+           'sumall', 'inner', 'mmin', 'mmax']
 
 
-def assign(f, v=0):
+def assign(f, RHS=0, options=None, name='asign', **kwargs):
     """
-    Assign a value to a Function.
+    Assign a list of RHS's to a list of Function's.
```

Comment:
nitpicking: we don't use apostrophes for plurals such as Functions

Reply:
perhaps it's now worth adding a couple examples?

Reply:
Yes, there will be + quite a few tests.